src/os/bluestore/BlueStore.h

```diff
@@ -188,6 +188,12 @@ enum {
   l_bluestore_gc_merged,
   //****************************************
 
+  // misc
+  //****************************************
+  l_bluestore_omap_iterator_count,
+  l_bluestore_omap_rm_keys_count,
+  //****************************************
```

Comment:
`l_bluestore_omap_rmkeys_count` would make it consistent
I do not see `l_bluestore_omap_rmkeys_ranges`. Omission?